app/views/content/a-day-in-the-life-of-a-teacher.md

```diff
@@ -17,79 +18,145 @@
+      classes: quote--indent
+    q-richard-art:
+      text: "The fact that I continue to love the subject I teach has meant that I have never grown weary of doing what I do."
+      name: "Richard, art teacher"
+      classes: quote--indent
+    q-mike-secondary:
+      text: "If you’ve had a hard day, there’s always someone there to listen - even if it’s just a two-minute natter or moan."
+      name: "Mike, secondary teacher"
+      classes: quote--indent
+    q-aaron-science:
+      text: "Teaching offers you something few other careers can offer you. To make a real difference to a large number of students lives for the better."
+      name: "Aaron, science teacher"
+      classes: quote--indent
+
+
 ---
+Teaching is a challenging but rewarding career that allows you to make a significant impact on pupils' lives. With no two days being the same, it’s an excellent choice for those who thrive in a dynamic and varied work environment.
+
+$q-richard-art$
+
+## A day in the life of a teacher
+
+### Morning
+
+#### 7:30am arrive at school
+
+Teachers often start their day early, arriving at school somwhere between 7:00am to 8:00am. The morning is usually spent preparing resources for the day ahead so that everything is organised for when the pupils arrive.
+
+
+#### 8:45am pupils arrive
+
+Depending on your role, you might be responsible for a form group or registering students. During this time, pupils typically engage in morning activities. It's also an opportunity to talk to students individually to check their wellbeing.
+
+
+#### 9:15am morning lessons 
+The structure of the school day will depend on your school, but often there are 2 lessons taught in the morning followed by a break. In total, there may be 5 hour-long lessons in the day. Each class could be around 30 pupils and will have its own dynamics as well as unique needs.
+
+$q-phil-secondary$
+
+
+#### 11:15am morning break
+Teachers are sometimes responsible for break-time duties. This involves supervising pupils for their safety but it is an opportunity to interact with pupils outside of the classroom and build relationships. 
+
+#### 11:35am lessons 
+
+After break, pupils return to lessons with renewed energy, which can be harnessed and channeled into their work. During this time, teachers usually teach two more lessons before lunch.
+$q-jessica-science$
+
+
+### Lunch
+
+Lunch is an opportunity for teachers to have a well-earned break and go to the staff room to talk about how the morning has been. Many teachers also use this as an opportunity to get ahead on their marking. By tackling some books during this time, it can reduce the amount of marking needed to be done in the afternoon.
+
+$q-mike-secondary$
+
+
+### Afternoon
+
+#### 2:15pm afternoon lessons 
+
+After lunch, there is often just one more lesson to teach. Schools sometimes work the timetable this way because after lunch pupils are generally more excitable making it harder to teach.  
```

Comment:
```suggestion
After lunch, there's often just one more lesson to teach. Schools sometimes work the timetable this way because after lunch pupils are generally more excitable making it harder to teach.  
```